ocaml-gen/derive/src/lib.rs

```diff
@@ -479,7 +479,16 @@ pub fn derive_ocaml_gen(item: TokenStream) -> TokenStream {
                         );
                     }
                 }
-                format!("{{ {} }} [@@boxed]", generics_ocaml.join("; "))
+
+                // See https://v2.ocaml.org/manual/attributes.html for boxing/unboxing
+                if generics_ocaml.len() == 1 {
+                    // Tell the OCaml compiler to not unbox records of size 1
+                    format!("{{ inner: {} }} [@@boxed]", generics_ocaml[0])
```

Comment:
I think you just need to remove the `inner:` here

Reply:
Done